RallyHereDebugTool/Source/Private/RHDTW_Session.cpp

```diff
@@ -1607,6 +1562,12 @@ void FRHDTW_Session::Do()
 			ImGui::EndTabItem();
 		}
 
+		if (pGIMatchmakingCache != nullptr && ImGui::BeginTabItem("Session Types", nullptr, ImGuiTabItemFlags_None))
```

Comment:
This is dependent on the local player session subsystem rather than the matchmaking catch, though I have a note on my todo list to move the storage to one of the global caches.